changelog/21.0/21.0.0/summary.md

```diff
@@ -199,10 +201,34 @@ KEYS mode analyzes the query structure without executing it, providing JSON outp
 
 These new VEXPLAIN modes enhance Vitess's query analysis capabilities, allowing for more informed decisions about sharding strategies and query optimization.
 
-### <a id="errant-gtid-vttablet"/>Errant GTID Detection on Vttablets
+### <a id="errant-gtid-vttablet"/>Errant GTID Detection on Vttablets</a>
 
 Vttablets now run an errant GTID detection logic before they join the replication stream. So, if a replica has an errant GTID, it will
 not start replicating from the primary. It will fail the call the set its replication source because of the errant GTID. This prevents us 
 from running into situations from which recovery is very hard.
 
 For users running with the vitess operator on kubernetes, this change means that the replicas with errant GTIDs will have broken replication and will report as unready. The users will need to manually clean up these errant replica tablets.
+
+### <a id="auto-replace-mysql-autoinc-with-seq"/>Automatically Replace MySQL auto_increment Clauses with Vitess Sequences</a>
+
+When migrating tables from an unsharded keyspace to a sharded one using the [VReplication `MoveTables` command](https://vitess.io/docs/reference/vreplication/movetables/), we now support
```

Comment:
This is too verbose for a summary. Can you please condense it into a short paragraph? No need to give all the background, just describe the feature briefly.

Reply:
In my mind if we add something to the summary we think it’s worth highlighting. At a minimum we should then describe what the thing is that we’re highlighting and explain why we think it’s worth highlighting. Which to me includes the context for what it is, what it is, why it’s important and how to use it. Two paragraphs is not a lot IMO and if you look at this and previous summary docs multiple paragraphs is more the norm than not. That being said, I will try and condense it and eliminate some content that isn’t strictly necessary for the stated objectives.

Reply:
I created a docs PR for this: https://github.com/vitessio/website/pull/1859

Once that's merged this can become a short paragraph that links to this for additional details: https://deploy-preview-1859--vitess.netlify.app/docs/21.0/reference/vreplication/movetables/#auto-increment-handling

Reply:
Updated in https://github.com/vitessio/vitess/pull/16860/commits/3ceb25362bf7a3b90fb9a24d49b28662e992a493